package/yast2-migration.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Oct  7 10:30:08 UTC 2016 - igonzalezsosa@suse.com
+
+- fixed "reboot" button after migration is finished
```

Comment:
NP: I'd add a technical detail behind this as the fix does not touch any code and you can "fix" it just by updating the yast2 package. Something like `(require yast2  >= 3.1.108.7 which provides the reboot functionality)`.